### PR TITLE
Fix: dispatching change event after new value is set.

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
@keithamus, Please review the PR.

### CODE REVIEW

File: src/text-expander-element.ts [hunk 1]
```diff
@@ -129,6 +129,8 @@ class TextExpander {
 
     this.input.value = beginning + value + remaining
 
+    this.input.dispatchEvent(new Event('change', {bubbles: true}))
```
Comment (keithamus) [inline src/text-expander-element.ts:132]: Can we please add a test that confirms this behaviour, that we can also use to validate and avoid regressions here.

Comment (Sachin-chaurasiya) [reply src/text-expander-element.ts:132]: @keithamus , I don't have experience in testing. can you please help me out here?

Comment (Sachin-chaurasiya) [reply src/text-expander-element.ts:132]: @keithamus, I have added some tests by looking at previous tests.

**Comment by Sachin-chaurasiya** [comment]
@keithamus, this will be same as `InputEvent('input')` and we need check for `text-expander-change` event for ` Event('input', {bubbles: true})` as well.

**Comment by Sachin-chaurasiya** [comment]
@keithamus , please review this PR.

File: src/text-expander-element.ts (hunk 1 above)
Comment (koddsson) [inline src/text-expander-element.ts:132]: The tests run green for me with this line removed so either the tests are wrong or the implementation.

**Review by koddsson — COMMENTED** [review]
Hey @Sachin-chaurasiya; the tests run fine for me when I remove the line that dispatches the `change` event.

Maybe you can add motivation for this change to the PR and double-check that this `change` event is being dispatched already by the browser.

**Comment by Sachin-chaurasiya** [comment]
Closing this PR as an Issue is different.

**Pull request closed without merging**